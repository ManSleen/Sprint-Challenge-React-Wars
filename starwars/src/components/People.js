import React from "react";
import Person from "./Person";

import { Card } from "semantic-ui-react";
import "../App.css";

const People = ({ people }) => {
  console.log(people);
  return (
    <div className="people-container">
      <Card.Group centered>
        {people.map(person => {
          return <Person key={person.name} person={person} />;
        })}
      </Card.Group>
    </div>
  );
};

export default People;
