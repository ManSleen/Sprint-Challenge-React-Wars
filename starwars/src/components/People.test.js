import React from "react";
import renderer from "react-test-renderer";

import People from "./People";

describe("<People />", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(<People />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
