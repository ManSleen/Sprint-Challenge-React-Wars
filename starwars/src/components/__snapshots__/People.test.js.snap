// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<People /> should match snapshot 1`] = `
<div
  className="people-container"
>
  <div
    className="ui centered cards"
  >
    <div
      className="ui active transition visible dimmer"
      onClick={[Function]}
    >
      <div
        className="content"
      >
        <div
          className="ui text loader"
        >
          Loading...
        </div>
      </div>
    </div>
  </div>
</div>
`;
